.travis.yml

```diff
@@ -3,9 +3,26 @@ sudo: false
 branches:
 only:
   - master
+
 python:
-  - "3.5"
-  - "3.6"
+  - 3.5
+  - 3.6
+env:
+    - PYSAL_PYPI=true PYSAL_PLUS=true
```

Comment:
It seems that `PYSAL_PLUS` is not specified later. I think it is safe to delete it.

Reply:
good catch. will update and push.